scaffolding-chef-inspec/tests/user-linux-inputs/inputs.yml

```diff
@@ -0,0 +1,2 @@
+foo_2: foo_2 from inputs
```

Comment:
Should this file still be here? If I'm understanding this right the yaml is set by the default.toml so I'm not sure this file is necessary.